modules/apps/calendar/calendar-web/src/main/java/com/liferay/calendar/web/internal/info/item/provider/CalendarBookingInfoItemFieldValuesProvider.java

```diff
@@ -84,6 +86,63 @@ calendarBooking, _getThemeDisplay())
 		}
 	}
 
+	/**
+	 * See {@link
+	 * com.liferay.calendar.internal.notification.NotificationTemplateContextFactory#_getCalendarBookingURL(
+	 * User, long)}
+	 */
+	protected String getCalendarBookingURL(CalendarBooking calendarBooking) {
+		try {
+			Company company = _companyLocalService.getCompany(
+				calendarBooking.getCompanyId());
+
+			String portalURL = company.getPortalURL(
```

Comment:
Hi @carolmariaabb, should portalURL be reordered?

Reply:
@linolaoj I updated the code. Thanks!